modules/@apostrophecms/schema/ui/apos/components/AposInputSlug.vue

```diff
@@ -68,10 +75,42 @@ export default {
       } else {
         return null;
       }
+    },
+    prefix () {
+      return this.field.prefix || '';
     }
   },
+  watch: {
+    following(newValue, oldValue) {
+      if (this.compatible(oldValue, this.next)) {
+        this.next = this.slugify(newValue);
+      }
+    }
+  },
+  mounted() {
+    this.debouncedCheckConflict = debounce(() => this.checkConflict(), 250);
```

Comment:
See above.

Reply:
To be more exact...

`this.checkConflict` is a method, but the debounced version can't be because it is created dynamically with `debounce-async`, which is a higher order function (a function that returns a new function).

Reply:
Again, arrow functions to the rescue, allowing us to have a lexical `this` without `const self = this;` boilerplate.